.github/workflows/test.yml

```diff
@@ -41,6 +41,8 @@ jobs:
         uses: ./.github/workflows/install-circt
         with:
           version: ${{ matrix.circt }}
+      - name: Install FileCheck
+        uses: ./.github/workflows/install-filecheck
```

Comment:
As a complete alternative, is there some Ubuntu package which provides FileCheck? Is there any way that we can avoid packaging FileCheck ourselves? (Is this a requirement because of Windows developers?)

Reply:
> As a complete alternative, is there some Ubuntu package which provides FileCheck?

Yeah we can just use llvm-15-tools, I'll switch to that for CI

> Is there any way that we can avoid packaging FileCheck ourselves? (Is this a requirement because of Windows developers?)

I'm not sure that there's a strict requirement, but AFAICT there is no project packaging FileCheck in a standalone and platform independent way which is why I created https://github.com/jackkoenig/FileCheck. We can point Windows developers to it if there are any, but no need to use it here! It's also nice for local development on my Mac although I of course can just build FileCheck myself.